CHANGELOG

```diff
@@ -1,3 +1,10 @@
+* Tue Mar 19 2019 Chris Tessmer <chris.tessmer@onyxpoint.com> - 3.14.1-0
+- Removed simplib's `deep_merge()` 3.x function
+  - NOTE: This is a bugfix and not a breaking change; simplib depends on
+    stdlib, which has long since provided a compatible `deep_merge()` function
+    in the same (global/Puppet 3.x) namespace, but with a superior
+    implementation.  For more details, see SIMP-6259.
```

Comment:
On isolated networks, I don't think referencing a JIRA ticket is helpful

Reply:
I don't think providing a reference ID is any less helpful than referring to a public URL under the same circumstances.  Both have precedent in our current changelogs.

In this case, providing the JIRA issue provided the most helpful balance I could find between either derailing the focus of the (already large) changelog entry with a deep comparison of implementation details, or just not explaining why removing an entire function did not constitute an API-breaking change.